bigbluebutton-html5/imports/ui/components/chat/component.jsx

```diff
@@ -51,6 +53,8 @@ const Chat = (props) => {
     lastTimeWindowValuesBuild,
   } = props;
 
+  const userSentMessage = UserSentMessageCollection.findOne({ userId: Auth.userID, sent: true });
```

Comment:
Why did you move this from container to component?

Reply:
To make sure the component updates even if all other props have the same value

Reply:
> Why did you move this from container to component?

To make sure the component updates even if all other props have the same value